src/utils/Errors.ts

```diff
@@ -0,0 +1,25 @@
+import { Error } from '../enums/Error';
+
+export const handlerError = (isError: Error) => {
+  switch (isError) {
+    case Error.RESET:
+      return '';
+    case Error.ADD:
+      return 'Unable to add a todo';
+    case Error.REMOVE:
+      return 'Unable to delete a todo';
+    case Error.UPDATE:
+      return 'Unable to delete a todo';
+    case Error.TITLE:
+      return 'Title can\'t be empty';
+    case Error.DATA:
+      return 'Data error';
+    case Error.USER:
+      return 'Check your USER_ID';
+    case Error.UPLOAD:
+      return 'Unable to upload a todos';
+
+    default:
+      return 'Something is wrong';
+  }
```

Comment:
You can do something like this:

Create an object with errors texts:

```
const Errors = {
  UPLOAD: "Unable to upload a todos"
  ...
}
```

Rename `isError` to `errorType`.
`retrurn Error[errorType as keyof type Error];`

For example:
`errorType = "UPLOAD";`
`Error[errorType as keyof type Error]` will be the same as `Error.UPLOAD`;

Reply:
This will help you avoid this big `switch case` that will grow and grow when you need to add new errors handle.